### DISCUSSION TITLE
Getting a field from a relationship table in automation

### DISCUSSION THREAD

**Discussion by JCV67** [Budibase/budibase#7711] [Help]
Hello,

I have a table A (reservations) with a relationship field to a table B (participants). I'm trying to create an automation to send a booking confirmation to the meeting creator (whose e-mail filed is in the table A) and to the participants (whose e-mail addresses are stored on the Table B)
Everything works except getting the e-mail addresses from table B.

![filter1](https://user-images.githubusercontent.com/104722906/189409286-9dfa4601-41c9-45b7-b06e-ada3e6540385.jpg)

Can you please help me
Thank you

**Comment by insomniacj93** [comment]
Boas! @JCV67 Good to see a fellow PT here!

I'm not very familiar with automations myself, but I believe the issue you're facing might have to do with the handlebars usage.

Here's the Budibase Doc's on data using automations, hopefully you can find your answer:
https://docs.budibase.com/docs/contextual-bindings#using-bindings

Wish i could help you further more, but i strongly believe someone else here will be able to guide you better on this process.

Best of Luck!

**Reply by JCV67** [reply]
Olá @insomniacj93 , goog to see you to. Yes my problem has to do with handlebars, I cannot extract data from a relationship table.
